common/static/common/js/discussion/views/discussion_thread_view.js

```diff
@@ -396,6 +404,7 @@
                     model: this.model,
                     mode: this.mode,
                     context: this.context,
+                    start_header: this.start_header,
```

Comment:
This file seems to have both a local variable start_header, and this.start_header. Can you explain?